src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/PropertiesManagerForJakarta.java

```diff
@@ -122,8 +122,19 @@ private MicroProfileJavaCompletionParams adapt(JakartaJavaCompletionParams param
 
     private JavaCursorContextResult adapt(org.eclipse.lsp4mp.commons.JavaCursorContextResult contextResult) {
         if (contextResult != null) {
-            var kind = contextResult.getKind();
-            return new JavaCursorContextResult(JavaCursorContextKind.forValue(kind.getValue()), contextResult.getPrefix());
+            return new JavaCursorContextResult(adapt(contextResult.getKind()), contextResult.getPrefix());
```

Comment:
Since the new `adapt()` method can return `null`, do we know if `null` is safe to pass as the `JavaCursorContextKind` parameter for the `JavaCursorContextResult` constructor?

Reply:
It is safe. It's just writing the value to a field and nothing else. The default constructor passes null values to that constructor already.